alert-policies/amazon-database-migration-service/LowFreeableMemory.yml

```diff
@@ -0,0 +1,36 @@
+name: Low Freeable Memory
+
+description: |+
+  This alert occurs when Freeable Memory bytes go below 100 for 5 minutes
```

Comment:
What is 100? Please specify the unit of measurement?

Reply:
Done. Please review.